app/Transformers/UserTransformer.php

```diff
@@ -31,10 +31,22 @@ class UserTransformer extends Transformer
      */
     public function transform(User $user)
     {
+        $role = $this->getUserRole($user);
         return [
             'id' => (int) $user->id,
             'name' => (string) $user->name,
             'email' => (string) $user->email,
+            'role' => (string) $role,
```

Comment:
Why do this (have a method to get the user's role, store that in a variable and assign that variable to the `role` field) when you could assign the following instead:
`$user->roles->first()?->name`

Reply:
You are right, it is better that way